enterprise/backend/test/metabase_enterprise/search/scoring_test.clj

```diff
@@ -18,67 +21,104 @@
       (is (= [1 3 2] (score [(item 1 "verified") (item 2 nil) (item 3 nil)]))))))
 
 (defn- ee-score
-  [search-string]
-  (fn [item]
-    (with-redefs [#_{:clj-kondo/ignore [:deprecated-var]} premium-features/enable-enhancements? (constantly true)]
-      (-> (scoring/score-and-result search-string item) :score))))
+  [search-string item]
+  (with-redefs [#_{:clj-kondo/ignore [:deprecated-var]}
+                premium-features/enable-enhancements? (constantly true)]
+    (-> (scoring/score-and-result search-string item) :score)))
 
 (defn- oss-score
-  [search-string]
-  (fn [item]
-    (with-redefs [#_{:clj-kondo/ignore [:deprecated-var]} premium-features/enable-enhancements? (constantly true)]
-      (-> (scoring/score-and-result search-string item) :score))))
+  [search-string item]
+  (with-redefs [#_{:clj-kondo/ignore [:deprecated-var]}
+                premium-features/enable-enhancements? (constantly false)]
```

Comment:
This typo caused me to make a bunch of changes in these tests! `oss-score` and `ee-score` were behaving the same way.